File: 1_starter/java/src/main/java/no/systek/graphqlworkshop/resolvers/PersonResolver.java
package no.systek.graphqlworkshop.resolvers;

import com.coxautodev.graphql.tools.GraphQLResolver;
import no.systek.graphqlworkshop.storage.Person;
import org.springframework.stereotype.Component;

@Component
public class PersonResolver implements GraphQLResolver<Person> {

  public String getDescription(Person person) {
    return String.format("%s, %s", person.getName(), person.getTitle());
  }
}

File: 1_starter/kotlin/src/main/kotlin/no/systek/graphqlworkshop/resolvers/PersonResolver.kt
package no.systek.graphqlworkshop.resolvers

import com.coxautodev.graphql.tools.GraphQLResolver
import no.systek.graphqlworkshop.storage.Person
import org.springframework.stereotype.Component

@Component
class PersonResolver : GraphQLResolver<Person> {
    fun getDescription(person: Person): String = String.format("%s, %s", person.name, person.title)
}
